packages/spectrum/src/additional/ArrayToolbar.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+  The MIT License
+
+  Copyright (c) 2017-2019 EclipseSource Munich
+  https://github.com/eclipsesource/jsonforms
+
+  Copyright (c) 2020 headwire.com, Inc
+  https://github.com/headwirecom/jsonforms-react-spectrum-renderers
+
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+
+import {
+  ActionButton,
+  Button,
+  Flex,
+  Heading,
+  Tooltip,
+  TooltipTrigger,
+  View,
+} from '@adobe/react-spectrum';
+import Add from '@spectrum-icons/workflow/Add';
+import AlertCircle from '@spectrum-icons/workflow/AlertCircle';
+import React from 'react';
+
+export interface ArrayLayoutToolbarProps {
+  label: string;
+  errors: string;
+  path: string;
+  addItem(path: string, data: any): () => void;
+  createDefault(): any;
+}
+export const ArrayLayoutToolbar = React.memo(
+  ({
+    label,
+    errors,
+    addItem,
+    path,
+    createDefault,
+  }: ArrayLayoutToolbarProps) => {
+    return (
+      <Flex direction='row' justifyContent='space-between'>
+        <Heading level={4}>{label}</Heading>
+        <View isHidden={errors.length === 0}>
+          <TooltipTrigger delay={0}>
+            <Button isQuiet variant='negative'>
+              <AlertCircle color='negative' />
+            </Button>
+            <Tooltip>{errors}</Tooltip>
+          </TooltipTrigger>
+        </View>
```

Comment:
Etwas unschön, dass die Fehler dann in der Mitte dargestellt werden:

![image](https://user-images.githubusercontent.com/66047/99245923-b9602c00-2804-11eb-8ae8-fa84d466ad1a.png)

Dem '+'-Button würde ich ein `margin-left: auto;` geben, auf dem Container das `justify-content: space-between;` rausnehmen. Auch hier kannst du ein `align-items: center;` auf dem Container machen, damit die Elemente schön mittig sind.

Reply:
das habe ich vom Material Design abgeschaut - aber ja - ich denke es macht nicht so sinn das Icon mit den Fehlern in der Mitte zu haben...